perf/istio-install/values-istio-sds-vault.yaml

```diff
@@ -0,0 +1,30 @@
+global:
+  controlPlaneSecurityEnabled: false
+
+  mtls:
+    # Default setting for service-to-service mtls. Can be set explicitly using
+    # destination rules or service annotations.
+    enabled: true
+
+  sds:
+    enabled: true
+    udsPath: "unix:/var/run/sds/uds_path"
+    useNormalJwt: true
+
+nodeagent:
+  enabled: true
+  image: node-agent-k8s
+  env:
+    # The IP address and the port number of a publicly accessible example Vault server.
+    CA_ADDR: "https://34.83.129.211:8200"
```

Comment:
Does this file have to repeat everything present in the main values.yaml?

You just enable and disable specific things right?

Reply:
This file has SDS and mTLS setting needed for the tests. These settings are explicately configured in this file to reduce the coupling with other values.yaml. The decoupling makes it less likely to have test breakage caused by the changes in other values.yaml.